packages/sanity/src/core/store/_legacy/kv/KVStore.ts

```diff
@@ -0,0 +1,39 @@
+import {merge, type Observable, Subject} from 'rxjs'
+import {filter, map, switchMap} from 'rxjs/operators'
+
+import {resolveBackend} from './backends/resolve'
+import {type KVStore, type KVStoreValue} from './types'
+
+/** @internal */
+export function createKVStore(): KVStore {
```

Comment:
On naming conventions: I followed a couple of different style guides on capitalizing `KV`:
- [Microsoft](https://learn.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/141e06ef(v=vs.71))
- [AirBNB](https://github.com/airbnb/javascript?tab=readme-ov-file#naming--Acronyms-and-Initialisms)

If it's too divergent from the rest of the codebase, please let me know.

Reply:
I am actually wondering about the word `KV` if it's too generic in the monorepo. Looking at the code I am wondering if it becomes too hard to find later. Maybe in studio it can be `createKeyValueStore` ?

Reply:
Would you want that change just here? Or also, for example,
`useKeyValueStore` in `packages/sanity/src/core/store/_legacy/datastores.ts`, as the type, and so on?

Reply:
I think there's value in keeping it consistent with the backend, so if we're establishing that "kv" is the name for it in the backend, I think we should call it `kv` studio side as well (I agree that we should generally try to avoid abbreviations unless they are very well established, and kv may be in a gray area here).

Reply:
Changed per feedback. Thank you!